core/src/test/scala/unit/kafka/admin/ReassignPartitionsUnitTest.scala

```diff
@@ -30,7 +30,7 @@ import org.apache.kafka.common.{Node, TopicPartition, TopicPartitionInfo, TopicP
 import org.junit.Assert.{assertEquals, assertFalse, assertThrows, assertTrue}
 import org.junit.function.ThrowingRunnable
 import org.junit.rules.Timeout
-import org.junit.{After, Assert, Before, Rule, Test}
+import org.junit._
```

Comment:
We should not use a wildcard imports here.

Reply:
Got it.